src/display/text_layer.js

```diff
@@ -114,6 +116,7 @@ class TextLayer {
       div: null,
       properties: null,
       ctx: null,
+      minFontSize: null,
```

Comment:
Could we not compute the value directly here instead?

(Note that the pre-existing `ctx` case is slightly "special", since that one depends on the `lang`-property which isn't available at this point.)